### PR TITLE
boards/nucleo-f722ze/Makefile.features: Add periph_can into Makefile.features

### PR DESCRIPTION
**Pull request by madokapeng**
### Contribution description
stm32f722ze MCU contains CAN 2.0B interface. Makefile.features for board nucleo-f722ze is missing this periph_can.

### Testing procedure
Have not done tests due to lack of the board.

### Issues/PRs references
Fix for https://github.com/RIOT-OS/RIOT/issues/15711

### CODE REVIEW

**Comment by fjmolinas** [comment]
Hi @madokapeng thanks for the contribution. Do you know someone that has the `BOARD` and could test? We avoid merging code that is untested, and in this case as you say you haven't tested CAN on that `BOARD`, there could be some configuration missing in `periph_conf.h` for example.

**Comment by fjmolinas** [comment]
Maybe @donsez has the `BOARD` and can test?

File: boards/nucleo-f722ze/Makefile.features [hunk 1]
```diff
@@ -8,6 +8,7 @@ FEATURES_PROVIDED += periph_rtt
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
 FEATURES_PROVIDED += periph_usbdev
+FEATURES_PROVIDED += periph_can
```
Comment (benpicco) [inline boards/nucleo-f722ze/Makefile.features:11]: You will also need to add this to `boards/nucleo-f722ze/Kconfig`

**Comment by benpicco** [comment]
You will also need up update `cpu/stm32/include/candev_stm32.h`

**Comment by madokapeng** [comment]
> Hi @madokapeng thanks for the contribution. Do you know someone that has the `BOARD` and could test? We avoid merging code that is untested, and in this case as you say you haven't tested CAN on that `BOARD`, there could be some configuration missing in `periph_conf.h` for example.

Understood. let me see if I could get one

**Comment by madokapeng** [comment]
> You will also need up update `cpu/stm32/include/candev_stm32.h`

I will address your comments. Thank you

**Comment by madokapeng** [comment]
ordered board. will re-create a PR once it has tested.
Close this one.

**Comment by donsez** [comment]
@fjmolinas: sorry, my board is in the hand of students in internship.

**Comment by fjmolinas** [comment]
Awesome thanks @madokapeng!

**Pull request closed without merging**